docs/data/sdks/typescript-browser/index.md

```diff
@@ -163,6 +164,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 });
 ```
 
+The following are the event info got tracked in the page view events.
```

Comment:
Maybe should move this to an include to make it easier to keep in sync in all locations.